user-workspace/src/App.overrides.tsx

```
@@ -0,0 +1,478 @@
+            let verificationResult: ReclaimVerification.Response;
+            switch (verificationMethod) {
+                case 'providerId':
+                    const params: ReclaimVerification.Request = {
+                        // appId: config.REACT_APP_RECLAIM_APP_ID ?? '',
+                        // secret: config.REACT_APP_RECLAIM_APP_SECRET ?? '',
+                        // providerId: inputText,
+                        // If you're overring sessions and providers, you can use any appId, secret, providerId
+                        appId: '00',
+                        secret: '00',
+                        providerId: '00',
+                    }
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)